command/agent/config.go

```diff
@@ -632,11 +640,13 @@ func DefaultConfig() *Config {
 		Telemetry: Telemetry{
 			StatsitePrefix: "consul",
 		},
-		SyslogFacility:      "LOCAL0",
-		Protocol:            consul.ProtocolVersion2Compatible,
-		CheckUpdateInterval: 5 * time.Minute,
-		AEInterval:          time.Minute,
-		DisableCoordinates:  false,
+		SyslogFacility:             "LOCAL0",
+		Protocol:                   consul.ProtocolVersion2Compatible,
+		CheckUpdateInterval:        5 * time.Minute,
```

Comment:
This seems really slow, given how cheap the work is. Any reason not to use like 5 seconds?